requirements.txt

```diff
@@ -2,3 +2,4 @@ junebug==0.1.14
 vxyowsup==0.1.8
 yowsup2==2.5.2
 vxaat==0.5.5
+vumi_msisdn_normalize_middleware
```

Comment:
Looks like we're pinning everything else - can you pin this too, to the latest version